modules/express-engine/schematics/install/files/__serverFileName@stripTsExtension__.ts

```diff
@@ -3,11 +3,27 @@ import 'zone.js/dist/zone-node';
 import { ngExpressEngine } from '@nguniversal/express-engine';
 import * as express from 'express';
 import { join } from 'path';
-
-import { AppServerModule } from './src/<%= stripTsExtension(main) %>';
 import { APP_BASE_HREF } from '@angular/common';
+import { addSSL } from './server-tools/add-ssl';
+import { ServerSettings } from './server-tools/server-settings.interface';
+
 import { existsSync } from 'fs';
 
+/** Must be imported before AppServerModule to allow DOM mocks to initialize */
+import './server-tools/windows-mocks-and-polyfills'
+
+import { AppServerModule } from './src/<%= stripTsExtension(main) %>';
+
+
+const serverSettings: ServerSettings = {
+  /** if you don't want to use SSL, switch this setting to false */
+  useSSL: true,
+  sslKey: '',
+  sslCert: '',
+  hostName: 'localHost',
+  port: Number.parseInt(process.env.PORT || '<%= serverPort %>' || '0', 10) || 4000,
```

Comment:
Why do we need all these new fallbacks? `serverPort` cannot be undefined hence anything after that is redundant.

```suggestion
  port: +process.env.PORT ?? <%= serverPort %>,
```